Automattic/wp-calypso
# Site Migration: Add 3 testimonials to upgrade plan screen for english locale

Related to https://github.com/Automattic/wp-calypso/issues/88970

## Proposed Changes

* These changes add 3 approved testimonials to the site migration upgrade plan screen for English locale only. The English locale check will be removed when the translations are ready.

<img width="792" alt="CleanShot 2024-04-09 at 09 39 34@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/79d7e1b4-9522-4121-8143-8e5dc02b9f7a">


## Testing Instructions



* Checkout branch or use live link.
* Navigate to `/sites?hosting-flow=true` and click `Migrate a site`.
* Go through the migration flow until you reach the upgrade plan screen.
* Switch to the English locale and verify the testimonials display correctly. Compare to the design: 5yhaC6umexdQLlLpCrCyEe-fi-2011_31943
* Switch to a different locale and verify the testimonials do not appear.

## Pre-merge Checklist
